### ISSUE TITLE
Inverse motors Rumble setting does not appear to apply or save correctly

### ISSUE THREAD

**Issue by shanebeeps** [schmaldeo/DS4Windows#56]
**Describe the bug**
Checking the Inverse motors checkbox in the Profiles setting does not appear to do anything different when testing motors, and when Applying, then Saving the profile, then going back into the profile, the checkbox is not checked. Seems like functionality may be bugged?

**Desktop (please complete the following information):**
Controller Make and Model: Moonlight virtual controller enabled with the virtual controller option in DS4W settings. HidHide set up to disable virtual controller from appearing in Windows otherwise.
OS: Windows 10 Pro 22H2
DS4Windows Version: 3.9.8

**Comment by schmaldeo** [comment]
Yes, it does indeed seem to be the case, because I rushed the release and didn't test it beforehand.

I sincerely apologise for both this and the delayed response, as I've been dealing with some stuff IRL that vastly limited the free time in which i could work on open source projects. I will fix it and release soon.

**Comment by schmaldeo** [comment]
[Here it is :))](https://github.com/schmaldeo/DS4Windows/releases/tag/v3.9.9)
